File: 758A-Holdiday of equality/holiday.py
def main ():
    n = int(input())
    citizens = []
    for _ in range (n):
        citizens.append(int(input()))
    
    richest_one = max(citizens)
    spent = 0

    for c in citizens:
        spent += richest_one - c
    
    print(spent)


main()

File: 758A-Holdiday of equality/holiday.cpp
#include <iostream>

using namespace std;

int main (void) {
    int n, max = 0,spent = 0;
    cin >> n;
    int citizens[n];
    for (int i = 0; i < n; i++){
        cin >> citizens[i];
        if (citizens[i] > max)
            max = citizens[i];
    } 
    for (int c : citizens)
        spent += max - c;
        
    cout << spent;
}